### PR TITLE
redo, processor(ticdc): set flushed resolvedTs when start table

### PR DESCRIPTION
**Pull request by CharlesCheung96**
### What problem does this PR solve?

Issue Number: close https://github.com/pingcap/tiflow/issues/9172

### What is changed and how it works?
set flushed resolvedTs when start table 

#### Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

### Release note 

```release-note
Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

If you don't think this PR needs a release note then fill it with `None`.
```

### CODE REVIEW

**Review by hicqu — APPROVED** [review]

File: cdc/processor/processor.go [hunk 1]
```diff
@@ -144,14 +145,7 @@ func (p *processor) AddTableSpan(
...
 				if p.redo.r.Enabled() {
-					var redoResolvedTs model.Ts
-					if barrier != nil {
-						redoResolvedTs = barrier.GlobalBarrierTs
-					} else {
-						stats := p.sinkManager.r.GetTableStats(span)
-						redoResolvedTs = stats.BarrierTs
-					}
-					p.redo.r.StartTable(span, redoResolvedTs)
+					p.redo.r.StartTable(span, checkpoint.ResolvedTs)
```
Comment (asddongmen) [inline cdc/processor/processor.go:148]: please add a comment to tell why we should dou this

**Review by asddongmen — APPROVED** [review]

File: cdc/processor/processor.go (hunk 1 above)
Comment (CharlesCheung96) [reply cdc/processor/processor.go:150]: Done, PTAL

**Pull request merged**